diff -up indi-1.8.2/cmake_modules/CMakeCommon.cmake.omv~ indi-1.8.2/cmake_modules/CMakeCommon.cmake
--- indi-1.8.2/cmake_modules/CMakeCommon.cmake.omv~	2019-12-20 17:01:41.873356314 +0100
+++ indi-1.8.2/cmake_modules/CMakeCommon.cmake	2019-12-20 17:01:48.709355982 +0100
@@ -39,7 +39,7 @@ IF (UNIX OR APPLE OR ANDROID)
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SEC_COMP_FLAGS}")
     SET(SEC_LINK_FLAGS "")
     IF (NOT APPLE AND NOT CYGWIN)
-        SET(SEC_LINK_FLAGS "${SEC_LINK_FLAGS} -Wl,-z,nodump -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now")
+        SET(SEC_LINK_FLAGS "${SEC_LINK_FLAGS} -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now")
     ENDIF ()
     IF (NOT ANDROID AND NOT APPLE)
         SET(SEC_LINK_FLAGS "${SEC_LINK_FLAGS} -pie")
