diff -uraN indi-1.8.5/cmake_modules/CMakeCommon.cmake omw-indi-1.8.5/cmake_modules/CMakeCommon.cmake
--- indi-1.8.5/cmake_modules/CMakeCommon.cmake	2020-04-19 16:19:26.000000000 +0200
+++ omw-indi-1.8.5/cmake_modules/CMakeCommon.cmake	2020-07-11 15:02:21.483531328 +0200
@@ -39,7 +39,7 @@
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${SEC_COMP_FLAGS}")
     SET(SEC_LINK_FLAGS "")
     IF (NOT APPLE AND NOT CYGWIN AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
-        SET(SEC_LINK_FLAGS "${SEC_LINK_FLAGS} -Wl,-z,nodump -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now")
+        SET(SEC_LINK_FLAGS "${SEC_LINK_FLAGS} -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now")
     ENDIF ()
     IF (NOT ANDROID AND NOT APPLE)
         SET(SEC_LINK_FLAGS "${SEC_LINK_FLAGS} -pie")
