diff -urN libindi-0.9.6-orig/CMakeLists.txt libindi-0.9.6-patched/CMakeLists.txt
--- libindi-0.9.6-orig/CMakeLists.txt	2012-11-20 07:39:14.000000000 +1100
+++ libindi-0.9.6-patched/CMakeLists.txt	2013-02-12 17:47:55.069159599 +1100
@@ -633,7 +633,7 @@
 
 add_executable(tutorial_one ${tutorialone_SRCS})
 
-target_link_libraries(tutorial_one indidriver m)
+target_link_libraries(tutorial_one indidriver m z)
 
 ########### Tutorial two ##############
 set(tutorialtwo_SRCS
@@ -643,7 +643,7 @@
 
 add_executable(tutorial_two ${tutorialtwo_SRCS})
 
-target_link_libraries(tutorial_two indidriver m)
+target_link_libraries(tutorial_two indidriver m z)
 
 ########### Tutorial three ##############
 if (CFITSIO_FOUND)
@@ -654,7 +654,7 @@
 
 add_executable(tutorial_three ${tutorialthree_SRCS})
 
-target_link_libraries(tutorial_three indidriver m)
+target_link_libraries(tutorial_three indidriver m z)
 endif (CFITSIO_FOUND)
 ########### Tutorial four ##############
 set(tutorialfour_SRCS
@@ -664,7 +664,7 @@
 
 add_executable(tutorial_four ${tutorialfour_SRCS})
 
-target_link_libraries(tutorial_four indidriver m)
+target_link_libraries(tutorial_four indidriver m z)
 
 ########### Tutorial five - dome driver ##############
 set(tutorialdome_SRCS
@@ -674,7 +674,7 @@
 
 add_executable(tutorial_dome ${tutorialdome_SRCS})
 
-target_link_libraries(tutorial_dome indidriver)
+target_link_libraries(tutorial_dome indidriver z)
 
 ########### Tutorial five - rain driver ##############
 set(tutorialrain_SRCS
@@ -684,7 +684,7 @@
 
 add_executable(tutorial_rain ${tutorialrain_SRCS})
 
-target_link_libraries(tutorial_rain indidriver)
+target_link_libraries(tutorial_rain indidriver z)
 
 ########### Client Tutorial ##############
 set(tutorialclient_SRCS
@@ -693,7 +693,7 @@
 
 add_executable(tutorial_client ${tutorialclient_SRCS})
 
-target_link_libraries(tutorial_client indiclient)
+target_link_libraries(tutorial_client indiclient z)
 #################################################################################
 
 install( FILES drivers.xml ${CMAKE_SOURCE_DIR}/drivers/focuser/indi_tcfs_sk.xml DESTINATION ${DATA_INSTALL_DIR})
